### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Database Properties Preview_Files_Keyboard: When navigating to under the Files tab "Database files" grid column headers are not accessible with any keyboard keys.

### ISSUE THREAD

**Issue by Raj-ghuge** [microsoft/azuredatastudio#25208]
**"Please do not close this bug. This bug should only be closed by Trusted Tester after verification."**

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHub Tags:

#A11yTCS; #A11yMAS; #A11ySev1; #DesktopApp; #WCAG2.1.1; #FTP; #SH_Azure Data Studio_Nov2023_Linux_Dec2023; #SQL Azure Data Studio; #Linux; #Keyboard;

### Environment Details:

Application Name: Azure Data Studio
OS Name: Ubuntu 22.04.2 LTS
Azure data studio 1.47.0

### Repro Steps:

1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Navigate by using TAB key till "Database" and press enter to expand the tree.
4. Navigate by using TAB key till any database and pressF10 to open context menu.
5. Navigate by using TAB key till "Properties Preview" and press enter.
6. Navigate by using TAB key till "Files" tab and press enter.
7. Navigate by using TAB key till "Database files" table and try to change column size.
8. Observe that, unable to change column size with keyboard keys.

### Actual results:

When navigating to under the Files tab "Database files" grid column headers are not accessible with any keyboard keys.
**Note: Issue seen throughout the application for similar type control.**

### Expected results:

When navigating to Files "Database files" grid column headers should be accessible with keyboard & also should be able to adjust the column header with keyboard.

### User Impact:

Keyboard only users will face difficulties if the focus does not go to column header and cannot resize the column header.

### Attachments:

[25208.zip](https://github.com/microsoft/azuredatastudio/files/13948822/25208.zip)

**Comment by corivera** [comment]
@Raj-ghuge This is a duplicate of another accessibility issue here: https://github.com/microsoft/azuredatastudio/issues/25132

**Comment by Raj-ghuge** [comment]
@corivera
https://github.com/microsoft/azuredatastudio/issues/25132 the bug is on Win32 platform and above bug is of Linux.

**Comment by corivera** [comment]
@Raj-ghuge resizing columns is provided via the keyboard shortcut Shift+Alt+S

**Comment by corivera** [comment]
@Raj-ghuge Can you verify that the issue is fixed using that shortcut? You can find the latest Ubuntu build of ADS [here](https://azuredatastudio-update.azurewebsites.net/latest/linux-deb-x64/stable).

**Comment by Raj-ghuge** [comment]
@corivera
The keyboard shortcut Shift+Alt+S is not working for "Database files" grid table not working.

**Comment by kisantia** [comment]
@Raj-ghuge the keyboard shortcut might be slightly different on linux. To find the key mapping
1. open the command palette (View->Command Palette):
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/b456dca3-0c67-4887-b7d2-47a8bd48a3ae)

2. Type "keyboard shortcut" and select "Preferences: Open Keyboard Shortcuts"
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/7bfbf104-6006-4071-82c0-bcefa8e88cb4)

3. Search for "grid.resizeColumn" and see the shortcut (ex: on my mac, it shows shift+option+s):
![image](https://github.com/microsoft/azuredatastudio/assets/31145923/a82646c4-392c-42e2-92f3-4b6eca714e6c)

**Comment by corivera** [comment]
@Raj-ghuge what's not working here? I saw the size prompt popped up at the top of the ADS window in your video clip. The column size is set there and then after hitting Enter the new size is applied to the column.

**Comment by Raj-ghuge** [comment]
Verified the issue in the environment -
Application Name: Azure Data Studio
OS Name: Ubuntu 22.04.2 LTS
Azure data studio 1.49.0
The issue is fixed, hence closing the bug. Please find attachment.
#Closed;
#Regressed:03-14-24

https://github.com/microsoft/azuredatastudio/assets/130441232/05fdfc27-e045-4036-ac75-dc451ac2d5ed
